.gitignore

```diff
@@ -127,6 +127,8 @@ dmypy.json
 # End of https://www.gitignore.io/api/python
 0
 config.backup.json
+config.json
```

Comment:
Ignoring the config.json will cause the non docker instance to error out due to not having a config file to pull from. This is a holdover requirement from when I was requiring certain settings on initialization. In the future I will create the config file programatically, but for now we have to keep it in.

Reply:
Ok, I removed it here as config.json is personal for every user. Your commit has it restored, so I'll set this on resolved.